lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Klasifikátorům chybí random state, takže výsledky v tabulce nesedí s textem. Také tam není nikde vysvětleno, co to je SVM a Dummy a proč používáme dva klasifikátory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>